<ui version="4.0" >
 <class>ClusterEditorClass</class>
 <widget class="QWidget" name="ClusterEditorClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ClusterEditor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title" >
         <string>Files</string>
        </property>
        <layout class="QGridLayout" name="gridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Fiber file:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="lineFiberFile" />
         </item>
         <item row="0" column="2" >
          <widget class="QPushButton" name="btnFiberFile" >
           <property name="maximumSize" >
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>Open</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Cluster file:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="lineClusterFile" />
         </item>
         <item row="1" column="2" >
          <widget class="QPushButton" name="btnClusterFile" >
           <property name="maximumSize" >
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>Open</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title" >
         <string>Major Bundles</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" >
         <item row="0" column="0" colspan="2" >
          <widget class="QListWidget" name="listMajorBundles" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Expanding" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize" >
            <size>
             <width>16777215</width>
             <height>300</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QPushButton" name="btnAssign" >
           <property name="text" >
            <string>Assign</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3" >
        <property name="title" >
         <string>Results</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3" >
         <item row="0" column="0" colspan="2" >
          <widget class="QTableWidget" name="tableResult" />
         </item>
         <item row="1" column="1" >
          <widget class="QPushButton" name="btnSave" >
           <property name="text" >
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <spacer name="horizontalSpacer_2" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="dispFrame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
